### DISCUSSION TITLE
Problems with Exposed Port

### DISCUSSION THREAD

**Discussion by compgeniuses** [outline/outline#7235] [Self Hosting]
I have built an outline app for Nethserver 8 here https://github.com/geniusdynamics/ns8-outline which uses podman and systemd to package a single container with multiple docker containers in it.

I have deifned everything correctly, however, when running the app, everything seems to run PErfeclty fine on the interface and logs, but accessing the web UI, i am unable to get access to the App.

I have exposed port 3000, and defined port 3000 on the env configs.
this is then exposed with traefik, but for some reason, i am just unable to get it working properly.

What could be the problem.

**Comment by tommoor** [comment]
> i am just unable to get it working properly.

You'll need to describe your issue in a bit more detail. It sounds like this is more of a networking issue than anything else – Outline just listens on the given port using standard NodeJS APIs.

**Reply by compgeniuses** [reply]
I doubt its a networking issue. generally all other apps i have built the same way. if the port just works when we expose th port of the app it just workss.

Must we define the port parameter in env?

**Comment by JetSetDoritos** [comment]
Did you ever find an issue? Everything seems to be in order for me but I just can not get this to connect.

**Reply by compgeniuses** [reply]
not yet, will re-attempt
